hw/ip/prim/pre_dv/prim_crc32/README.md

```diff
@@ -0,0 +1,18 @@
+CRC32 Calculator Testbench
+==========================
+
+This is a primitive testbench to check the basic functionality of the CRC32
+primitive. It is not intended as a full verification environment.
+
+It is build via fusesoc (from repository root)
+
+  ```sh
+  fusesoc --cores-root=. run --target=sim --setup --build lowrisc:prim:crc32_sim
+  ./build/lowrisc_prim_crc32_sim_0/sim-verilator/Vprim_crc32_sim
+  ```
+
+The `expected_out.py` python script generates the expected output. This is
+simply a dump of expected CRC values as test data is fed in. This is chcked by
+eye, there is no automated checking.
```

Comment:
Does that mean that the Verilator testbench does not check expected vs actual responses?
Would it make sense to add basic checking (just read in the expected_out values and do a comparison in the TB), or alternatively dump the actual responses and diff the files?

Reply:
I was just trying to save a bit of time given it's in a `pre_dv` so not meant to be an actual testbench. Still a quick diff run by a script (like we do with the OTBN smoke test) should be easy enough.